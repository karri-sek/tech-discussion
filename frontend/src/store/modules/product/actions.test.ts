import {
  addToCartRequestAction
} from "./actions";

describe('Product actions', () => {
  it('Add to cart action', () => {
    expect(addToCartRequestAction()).toMatchSnapshot();
  });
});
