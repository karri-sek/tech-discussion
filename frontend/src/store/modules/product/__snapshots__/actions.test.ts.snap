// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product actions Add to cart action 1`] = `
Object {
  "product": undefined,
  "type": "ADD_TO_CART_REQUEST",
}
`;
